ranger-discovery-bundle/src/main/java/io/appform/ranger/discovery/bundle/id/formatter/Base36IdFormatter.java

```diff
@@ -29,6 +29,11 @@ public Base36IdFormatter(IdFormatter idFormatter) {
         this.idFormatter = idFormatter;
     }
 
+    @Override
+    public int getType() {
```

Comment:
This should throw an error given that this method would not be called at any point of time ?